sfge/src/main/java/com/salesforce/graph/ops/expander/PathExpansionRegistry.java

```diff
@@ -73,6 +73,19 @@ private static class ApexPathExpanderRegistryData extends RegistryData<ApexPathE
         ApexPathExpanderRegistryData() {
             pathExpansionLimit = SfgeConfigProvider.get().getPathExpansionLimit();
             isPathExpansionLimitSet = pathExpansionLimit > 0;
+
+            if (LOGGER.isInfoEnabled()) {
```

Comment:
New log line we'll print to surface the limit numbers and what heap space this is based on.